src/confluent_kafka/src/Consumer.c

```diff
@@ -1356,17 +1479,67 @@ static void Consumer_rebalance_cb (rd_kafka_t *rk, rd_kafka_resp_err_t err,
 		}
 	}
 
+        if (!strcmp(rd_kafka_rebalance_protocol(rk), "COOPERATIVE")) {
+                if (self->u.Consumer.rebalance_assigned) {
+                        assign_errstr = "assign() must not be called during a "
```

Comment:
This already enforced by librdkafka, don't need to do it here.

Reply:
... yep. whoever implemented that was a thinker.